components/atoms/icon.tsx

```diff
@@ -0,0 +1,10 @@
+import classNames from '../../utils/classes'
+import Img from './img'
+
+export default function Icon ({ src, alt, className = '' }) {
+  return (
+    <div className={classNames('flex items-center justify-center rounded-full', className)}>
+      <Img src={src} alt={alt} width={100} height={100} />
```

Comment:
I would prefer to see width and height with variables (you can set default values to 100 to them tho)